.github/scripts/devcloud-test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
commit "[[UR][Tests] Print all CTS outputs (from gtest) in CI](https://github.com/oneapi-src/unified-runtime/pull/832/commits/31127d92d8a23cfb960f195bb441f9ede7225bdf)" seems redundant

Reply:
Done